distribution/docs/src/main/resources/content/_managing/_securing/managing-keystores.adoc

```diff
@@ -9,9 +9,16 @@
 
 Certificates, and sometimes their associated private keys, are stored in keystore files.
 ${branding} includes two default keystore files, the server key store and the server trust store.
-The server keystore holds the certificates and private keys that ${branding} uses to identify itself to other nodes on the network.
+The server keystore holds ${branding}'s certificate and private key.
+It will also hold the certificates of other nodes whose signature ${branding} will accept.
```

Comment:
this sounds like a truststore function

Reply:
No, the trust store is only meant to contain certs or ca certs we'll trust for PKI login.